assets/src/design-system/components/input/index.js

```diff
@@ -99,6 +122,11 @@ export const Input = ({
         hasError={hasError}
         {...props}
       />
+      {Icon && (
+        <IconContainer data-testid="input-icon">
+          <Icon aria-hidden />
+        </IconContainer>
```

Comment:
We could add `pointer-events: none;` to the styling of this container so that when it is clicked the input will receive the event. However, if there's a plan to add click events to the icons then maybe this is not the approach we should take.

Reply:
i think i like `pointer-events: none` - I don't see a need to have the icon on basic inputs do anything outside of what the input would be doing on it's own. The search case is different, but that's why it has its own thang. If the use case changes we can adapt :)